Makefile.am

```diff
@@ -110,6 +110,7 @@ libvkd3d_la_SOURCES = \
 	libs/vkd3d/command.c \
 	libs/vkd3d/device.c \
 	libs/vkd3d/meta.c \
+	libs/vkd3d/platform.c \
```

Comment:
New headers should also be added to Makefile.am

Reply:
Wack. Okay.

Reply:
We're missing the one for vkd3d_atomic.h in that case...

Not sure why the headers are included in the autotools sources given they're just going to be ignored but... I'll add another commit to make this consistent anyway

Reply:
Done.

Reply:
Ye, not sure either. Just following convention. It makes sense if there is some magical IDE tooling with automake (which I doubt).